src/Auxiliary/Aux_Check_Parameter.cpp

```diff
@@ -247,8 +259,10 @@ void Aux_Check_Parameter()
    if ( ! OPT__FIXUP_RESTRICT )
       Aux_Error( ERROR_INFO, "must enable OPT__FIXUP_RESTRICT for BITWISE_REPRODUCIBILITY !!\n" );
 
+#  ifdef LOAD_BALANCE
```

Comment:
I think `OPT__SORT_PATCH_BY_LBIDX` should also be applied when disabling `LOAD_BALANCE`.